jira/client.py

```diff
@@ -196,7 +197,7 @@ class JIRA(object):
     AGILE_BASE_URL = GreenHopperResource.AGILE_BASE_URL
 
     def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=None, kerberos=False,
-                 validate=False, get_server_info=True, async=False, logging=True, max_retries=3, proxies=None):
+                 validate=False, get_server_info=True, async=False, max_retries=3, proxies=None):
         """Construct a JIRA client instance.
```

Comment:
-1
I don't think this flag actually does anything useful; it only protects one single logging call in `create_project()`, and it's not documented, so I really doubt anyone is using it. I really don't think it's worth preserving for the sake of maintaining backwards compatibility.

PS: @ssbarnea I [asked for your input](https://github.com/pycontribs/jira/pull/308#discussion_r92927997) on this back in December! Don't you monitor mentions?